docs(pipeline): add how to manage copilot version for pipeline (#3464)
aws/copilot-cli: site/content/docs/concepts/pipelines.en.md

![Your completed CodePipeline](https://user-images.githubusercontent.com/828419/71861318-c7083980-30aa-11ea-80bb-4bea25bf5d04.png)

!!! info
    If you have selected a GitHub or Bitbucket repository, Copilot will help you connect to your source code with [CodeStar Connections](https://docs.aws.amazon.com/dtconsole/latest/userguide/welcome-connections.html). You will need to install the AWS authentication app on your third-party account and update the connection status. Copilot and the AWS Management Console will guide you through these steps.

### Step 6: Manage Copilot Version for Your Pipeline (optional)

> After creating your pipeline, you can manage the version of Copilot used by your pipeline by updating the `buildspec.yml` as following:

Reviewer: ```suggestion
After creating your pipeline, you can manage the version of Copilot used by your pipeline by updating the `buildspec.yml` as follows:
```

Reviewer: Do we plan to update this with each release? If not, maybe something like: 
```suggestion
After creating your pipeline, you can manage the version of Copilot used by your pipeline by updating the following lines of your `buildspec.yml` to the latest version:
```

> After creating your pipeline, you can manage the version of Copilot binary used by your pipeline by updating the `buildspec.yml` as following:

Reviewer 2: nit
```suggestion
After creating your pipeline, you can manage the version of Copilot used by your pipeline by updating the `buildspec.yml` as following:
```
